less/services.less

```diff
@@ -0,0 +1,157 @@
+.services {    
+    background-color: white;
+    max-width: 100vw;
+    margin: 0 auto;
+    height: auto;
+    // position: relative;
+    .header-image-container {
+        position: relative;
+        text-align: center;
+        max-width: 100%;
+        h1 {
```

Comment:
The Services h1 text looks great especially on mobile! I love how you made it take up the entire header image 🙌